systems.csv

```diff
@@ -345,13 +344,14 @@ FR,Dott Paris,Paris,dott-paris,https://ridedott.com,https://gbfs.api.ridedott.co
 FR,Dott Tignes,Tignes,dott-tignes,https://ridedott.com,https://gbfs.api.ridedott.com/public/v2/tignes/gbfs.json,
 FR,Dott Val-d’isere,Val-d’isere,dott-val-d’iser,https://ridedott.com/,https://gbfs.api.ridedott.com/public/v2/val-d%E2%80%99isere/gbfs.json,
 FR,LE vélo STAR,Rennes,le_velo_star,https://www.star.fr/le-velo/nos-offres/vls/,https://eu.ftp.opendatasoft.com/star/gbfs/gbfs.json,
+FR,Levélo,Marseille,levelo_inurba_marseille,https://levelo.ampmetropole.fr,https://api.omega.fifteen.eu/gbfs/2.2/marseille/en/gbfs.json?&key=MjE0ZDNmMGEtNGFkZS00M2FlLWFmMWItZGNhOTZhMWQyYzM2,

```

Comment:
This format with a key would be a first in systems.csv I believe

Reply:
According to the files listed, the auto-discovery URL is https://api.omega.fifteen.eu/gbfs/2.2/marseille/en/gbfs.json and users need a token to access the data. Should we include the token in the last column instead?

Reply:
I added 2 commits to this PR to make this change ☝️